database/buildSchema/22_application_response.sql

```diff
@@ -1,14 +1,14 @@
 -- application response
 CREATE TYPE public.application_response_status AS ENUM (
-    'Draft',
-    'Submitted'
+    'DRAFT',
+    'SUBMITTED'
 );
 
 CREATE TABLE public.application_response (
     id serial PRIMARY KEY,
     template_element_id integer REFERENCES public.template_element (id),
     application_id integer REFERENCES public.application (id),
-    "status" public.application_response_status DEFAULT 'Draft',
+    status public.application_response_status DEFAULT 'SUBMITTED',
```

Comment:
Why did you change the default to "Submitted"?

Reply:
sorry, thanks for noticing